### PR TITLE
[11.0][ADD] partner_multi_name module

### PR DESCRIPTION
**Pull request by Capriatto**
This is our first contribution, all suggestions are welcome:

This module extends the "partner_second_lastname" functionality to have a middle name or other names in a separated way.

    Use Case:
    Partner Second Last Name: <Johannes Chrysostomus>  <Theophilus> <Mozart>
    Partner Other Names: <Johannes><Chrysostomus Wolfgangus>< Theophilus><Mozart>

### CODE REVIEW

File: partner_other_names/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+
+class ResPartner(models.Model):
+    """Adds other names."""
+    _inherit = "res.partner"
+
+    othernames = fields.Char("Other Names")
+
+    @api.model
+    def _get_computed_name(self, firstname, othernames, lastname, lastname2):
```
Comment (joao-p-marques) [inline partner_other_names/models/res_partner.py]: You could add default argument values here to increase resilience.
```suggestion
    def _get_computed_name(self, firstname=None, othernames=None, lastname=None, lastname2=None):
```

**Review by NL66278 — CHANGES_REQUESTED** [review]
Module name must be changed to not contain plural. See also corresponding review for 12.0 version.

File: partner_other_names/models/res_partner.py (hunk 1 above)
Comment (joao-p-marques) [reply partner_other_names/models/res_partner.py]: You didn't change this...

**Review by joao-p-marques — APPROVED** [review]

File: partner_other_names/models/res_partner.py (hunk 1 above)
Comment (Capriatto) [reply partner_other_names/models/res_partner.py]: You're right. We already did.

**Comment by Capriatto** [comment]
> Module name must be changed to not contain plural. See also corresponding review for 12.0 version.

@NL66278 is still necessary the "needs fixing" tag?

File: partner_multi_name/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (NL66278) [inline partner_multi_name/.vscode/settings.json:1]: This file should not be part of the PR

**Comment by NL66278** [comment]
It would be good to add some unit tests...

File: partner_multi_name/.vscode/settings.json (hunk 2 above)
Comment (grottas) [reply partner_multi_name/.vscode/settings.json:1]: @alejo-code @JoanMarin FYI
